hits_cpp/hits.cpp

```diff
@@ -91,8 +91,12 @@ void memeHits(Meme& meme, AlphabetMap& alphabet, SequencesMap& sequences, int& h
             }
             counter++;
             if (isHit(meme, alphabet, sequencesTypesIter->first, *sequencesIter, verbose)) {
-                meme.addHitSequence(*sequencesIter, isOutputSequences);
-                hits++;
+                if (isUseUniqueRpm){
+                    meme.addHitSequence(*sequencesIter, isOutputSequences, sequncesRpm.find(*sequencesIter)->second);
+                } else {
+                    meme.addHitSequence(*sequencesIter, isOutputSequences);
```

Comment:
Should have "1" as third parameter for better readability and symmetry.
If `...->second` is 1 in this case then condition is not required

Reply:
...->second in sequncesRpm does not return 1 when it is shuffles.